<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QcepImagePlotWidgetDialog</class>
 <widget class="QDialog" name="QcepImagePlotWidgetDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>603</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Image Display Options...</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="1">
      <widget class="QCheckBox" name="m_Y2AxisVis">
       <property name="toolTip">
        <string>2nd Y axis visible?</string>
       </property>
       <property name="text">
        <string>2nd Y Axis Visible</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QCheckBox" name="m_XAxisVis">
       <property name="toolTip">
        <string>X axis visible?</string>
       </property>
       <property name="text">
        <string>X Axis Visible</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QCheckBox" name="m_DisplayROI">
       <property name="toolTip">
        <string>Display ROI Borders</string>
       </property>
       <property name="text">
        <string>Display ROIs</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QCheckBox" name="m_YAxisLog">
       <property name="toolTip">
        <string>Log Y axis?</string>
       </property>
       <property name="text">
        <string>Y Axis Log</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QCheckBox" name="m_X2AxisVis">
       <property name="toolTip">
        <string>2nd X axis visible?</string>
       </property>
       <property name="text">
        <string>2nd X Axis Visible</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QCheckBox" name="m_X2AxisLog">
       <property name="toolTip">
        <string>Log 2nd X axis?</string>
       </property>
       <property name="text">
        <string>2nd X Axis Log</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QCheckBox" name="m_YAxisVis">
       <property name="toolTip">
        <string>Y axis visible?</string>
       </property>
       <property name="text">
        <string>Y Axis Visible</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QCheckBox" name="m_XAxisLog">
       <property name="toolTip">
        <string>Log X axis?</string>
       </property>
       <property name="text">
        <string>X Axis Log</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QCheckBox" name="m_DisplayImage">
       <property name="toolTip">
        <string>Display image data</string>
       </property>
       <property name="text">
        <string>Display Image</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QCheckBox" name="m_Y2AxisLog">
       <property name="toolTip">
        <string>Log 2nd Y axis?</string>
       </property>
       <property name="text">
        <string>2nd Y Axis Log</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Plot Legend Position:</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QCheckBox" name="m_DisplayOverflow">
       <property name="toolTip">
        <string>Display Overflow</string>
       </property>
       <property name="text">
        <string>Display Overflow</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Color Map</string>
       </property>
      </widget>
     </item>
     <item row="13" column="0" colspan="2">
      <widget class="QStackedWidget" name="m_DisplayParmsStack">
       <property name="currentIndex">
        <number>2</number>
       </property>
       <widget class="QWidget" name="page">
        <layout class="QGridLayout" name="gridLayout_15">
         <item row="2" column="0">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Min (%):</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="1" colspan="2">
          <widget class="QDoubleSpinBox" name="m_DisplayMinimumPct">
           <property name="toolTip">
            <string>Min percentage</string>
           </property>
           <property name="keyboardTracking">
            <bool>false</bool>
           </property>
           <property name="suffix">
            <string> %</string>
           </property>
           <property name="minimum">
            <double>-20.000000000000000</double>
           </property>
           <property name="maximum">
            <double>120.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>Max (%):</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="3" column="1" colspan="2">
          <widget class="QDoubleSpinBox" name="m_DisplayMaximumPct">
           <property name="toolTip">
            <string>Max percentage</string>
           </property>
           <property name="keyboardTracking">
            <bool>false</bool>
           </property>
           <property name="suffix">
            <string> %</string>
           </property>
           <property name="minimum">
            <double>-20.000000000000000</double>
           </property>
           <property name="maximum">
            <double>120.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QPushButton" name="m_Display_5pct">
           <property name="text">
            <string>5% Range</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QPushButton" name="m_Display_10pct">
           <property name="text">
            <string>10% Range</string>
           </property>
          </widget>
         </item>
         <item row="4" column="2">
          <widget class="QPushButton" name="m_Display_100pct">
           <property name="text">
            <string>100% Range</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0" colspan="3">
          <spacer name="verticalSpacer_7">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>88</width>
             <height>58</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page_3">
        <layout class="QGridLayout" name="gridLayout_18">
         <item row="2" column="2">
          <widget class="QPushButton" name="m_Display_99pctle">
           <property name="text">
            <string>99% Percentile</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QPushButton" name="m_Display_95pctle">
           <property name="text">
            <string>95% Percentile</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1" colspan="2">
          <spacer name="verticalSpacer_9">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>70</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="2" column="0">
          <widget class="QPushButton" name="m_Display_90pctle">
           <property name="text">
            <string>90% Percentile</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_26">
           <property name="text">
            <string>Max (Percentile):</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_25">
           <property name="text">
            <string>Min (Percentile):</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" colspan="2">
          <widget class="QDoubleSpinBox" name="m_DisplayMinimumPctle">
           <property name="toolTip">
            <string>Min percentile</string>
           </property>
           <property name="keyboardTracking">
            <bool>false</bool>
           </property>
           <property name="suffix">
            <string> %</string>
           </property>
           <property name="minimum">
            <double>-20.000000000000000</double>
           </property>
           <property name="maximum">
            <double>120.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="1" column="1" colspan="2">
          <widget class="QDoubleSpinBox" name="m_DisplayMaximumPctle">
           <property name="toolTip">
            <string>Max percentile</string>
           </property>
           <property name="keyboardTracking">
            <bool>false</bool>
           </property>
           <property name="suffix">
            <string> %</string>
           </property>
           <property name="minimum">
            <double>-20.000000000000000</double>
           </property>
           <property name="maximum">
            <double>120.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page_2">
        <layout class="QGridLayout" name="gridLayout_19">
         <item row="1" column="1">
          <widget class="QDoubleSpinBox" name="m_DisplayMaximumVal">
           <property name="toolTip">
            <string>Max value</string>
           </property>
           <property name="keyboardTracking">
            <bool>false</bool>
           </property>
           <property name="suffix">
            <string/>
           </property>
           <property name="minimum">
            <double>-1000000000.000000000000000</double>
           </property>
           <property name="maximum">
            <double>1000000000.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string>Maximum:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>Minimum:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QPushButton" name="m_AutoRange">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="font">
            <font>
             <pointsize>8</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Auto Range</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QDoubleSpinBox" name="m_DisplayMinimumVal">
           <property name="toolTip">
            <string>Min value</string>
           </property>
           <property name="keyboardTracking">
            <bool>false</bool>
           </property>
           <property name="suffix">
            <string/>
           </property>
           <property name="minimum">
            <double>-1000000000.000000000000000</double>
           </property>
           <property name="maximum">
            <double>1000000000.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="3" column="0" colspan="2">
          <spacer name="verticalSpacer_10">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QComboBox" name="m_DisplayColorMap">
       <property name="toolTip">
        <string>Choose the color map for image display</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QCheckBox" name="m_MaintainAspectRatio">
       <property name="toolTip">
        <string>Try to maintain the aspect ratio of the image as it is zoomed and resized</string>
       </property>
       <property name="text">
        <string>Maintain Aspect Ratio</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="m_LegendPosition">
       <property name="toolTip">
        <string>Plot Legend Position</string>
       </property>
       <item>
        <property name="text">
         <string>Left</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Right</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Bottom</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Top</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QCheckBox" name="m_InterpolatePixels">
       <property name="toolTip">
        <string>Gives a smoother display when zoomed in</string>
       </property>
       <property name="text">
        <string>Interpolate Pixel Values</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QCheckBox" name="m_DisplayMask">
       <property name="toolTip">
        <string>Display Mask</string>
       </property>
       <property name="text">
        <string>Display Mask</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Overflow Level:</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QSpinBox" name="m_OverflowLevel">
       <property name="toolTip">
        <string>overflow (saturation) level (0..65535)</string>
       </property>
       <property name="maximum">
        <number>65536</number>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QCheckBox" name="m_DisplayImageLog">
       <property name="toolTip">
        <string>Logarithmic Intensity Scaling</string>
       </property>
       <property name="text">
        <string>Log Scaling</string>
       </property>
      </widget>
     </item>
     <item row="11" column="0" colspan="2">
      <widget class="QComboBox" name="m_DisplayScalingMode">
       <property name="toolTip">
        <string>Image display intensity scaling mode</string>
       </property>
       <item>
        <property name="text">
         <string>Display based on percentages of data range</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Display based on percentiles of pixel counts</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Display based on abs. values</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>QcepImagePlotWidgetDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>246</x>
     <y>440</y>
    </hint>
    <hint type="destinationlabel">
     <x>438</x>
     <y>349</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>QcepImagePlotWidgetDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>178</x>
     <y>439</y>
    </hint>
    <hint type="destinationlabel">
     <x>437</x>
     <y>283</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
